### PR TITLE
Raincatch 537 refactor result module

### PR DESCRIPTION
**Pull request by nialldonnellyfh**
# Motivation

This PR removes the `angular` functionality from the result module.

The result module does not have any UI functionality other than to provide the `resultSync` service.

Given that this service is no longer used in any of the mobile applications, it can be removed.

### CODE REVIEW

**Comment by witmicko** [comment]
LGTM

File: lib/client/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var mediatorSubscribers = require('./mediator-subscribers');
+
+/**
+ * Initialisation of the workorder core module.
+ * @param {Mediator} mediator
+ */
+module.exports = function(mediator) {
+
+  //Initialising the subscribers to topics that the module is interested in.
+  return mediatorSubscribers.init(mediator);
```
Comment (wtrocki) [inline lib/client/index.js]: Most of the modules using pattern where client is passed as second parameter. In this particular case client is static, but I think we should use the same patterns across the board to simplify refactoring.

**Review by wtrocki — APPROVED** [review]
Reviewed.

File: lib/client/index.js (hunk 1 above)
Comment (nialldonnellyfh) [reply lib/client/index.js]: Yes, good point. I will change.

**Pull request merged**